modules/javafx.controls/src/main/java/javafx/scene/control/SkinBase.java

```diff
@@ -206,39 +209,112 @@ protected final void consumeMouseEvents(boolean value) {
 
 
     /**
-     * Subclasses can invoke this method to register that they want to listen to
-     * property change events for the given property. Registered {@link Consumer} instances
-     * will be executed in the order in which they are registered.
-     * @param property the property
-     * @param consumer the consumer
+     * Registers an operation to perform when the given {@code observable} sends a change event.
+     * Does nothing if either {@code observable} or {@code operation} are {@code null}.
+     * If multiple operations are registered on the same observable, they will be performed in the
+     * order in which they were registered.
+     *
+     * @param observable the observable to observe for change events, may be {@code null}
+     * @param operation the operation to perform when the observable sends a change event,
+     *  may be {@code null}
```

Comment:
This method is missing the `@since 9` javadoc tag. This is a preexisting bug; this method was missed when [JDK-8135312](https://bugs.openjdk.java.net/browse/JDK-8135312) was fixed. It's tempting to fold it into this RFE, since you are rewriting the API docs for this method, but I'll file a separate bug instead.